src/main/webapp/resources/js/pages/UserGroupsPage/components/UserGroupMembersTable.jsx

```diff
@@ -35,21 +34,6 @@ export default function UserGroupMembersTable({
   groupId,
   updateTable,
 }) {
-  const { roles } = useContext(UserGroupRolesContext);
-
-  /**
-   * Remove a member from the group, and then refresh the table.
-   * @param user
-   * @returns {void | Promise<AxiosResponse<*>>}
-   */
-  function removeMember(user) {
-    return removeMemberFromUserGroup({ groupId, userId: user.id }).then(
-      (message) => {
-        updateTable();
-        return message;
-      }
-    );
-  }
 
```

Comment:
A question related to this file. Should it be allowed for a group to be left without an owner? Testing this out on the group members table let's me change the owner to a member (currently only the one member is a part of the group)?

Reply:
So... I cannot reproduce this.

Reply:
Oh, if the admin creates the group. They can change their own status.  Otherwise this does not happen.  Interesting.

Reply:
The method to assess this is completely screwed, trying to fix it now

Reply:
Fixed in 3a15ed6